### PR TITLE
vishala hotfix modal pop up scroll issue

### PR DESCRIPTION
**Pull request by Vishala09**
# Description
Fixes scroll issue of modal pop-up

## Main changes explained:
- Removed some css related to modal

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Please test if the modal anywhere in the website is easily scrollable

## Screenshots or videos of changes:

[Untitled_ Dec 06,2023 5_33 PM.webm](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/65274029/bf5d3cc3-2112-4fbd-83ce-121ee265a8f0)

### CODE REVIEW

File: src/components/BMDashboard/MaterialsList/MaterialsTable.jsx [hunk 1]
```diff
@@ -45,7 +45,9 @@ export default function MaterialsTable({ filteredMaterials }) {
   const sortingAsc = columnName => {
     let sorted = [];
     if (columnName === 'ProjectName') {
-      sorted = [].concat(...sortedData).sort((a, b) => (a.project.name >= b.project.name ? 1 : -1));
+      sorted = []
+        .concat(...sortedData)
+        .sort((a, b) => (a.project?.name >= b.project?.name ? 1 : -1));
```
Comment (stephan-noel) [inline src/components/BMDashboard/MaterialsList/MaterialsTable.jsx:50]: **nit**: Could [toSorted](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted) be clearer and more performant?  Not sure what browser versions we target.

File: src/components/BMDashboard/MaterialsList/SelectForm.jsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ export default function SelectForm({ materials, setSelectedProject, setSelectedM
   // create selectable projects
   let projectsSet = [];
   if (materials.length) {
-    projectsSet = [...new Set(materials.map(mat => mat.project.name))];
+    projectsSet = [...new Set(materials.map(mat => mat.project?.name))];
```
Comment (stephan-noel) [inline src/components/BMDashboard/MaterialsList/SelectForm.jsx:7]: **nit**:  Would `uniqueProjects` or similar be a better name?  It's actually an array and I feel like this is  tending towards Hungarian notation.

I see others like materials are doing similar.

File: src/components/Projects/Members/Member/Member.test.jsx [hunk 3]
```diff
@@ -77,38 +86,42 @@ describe('Member Component', () => {
...
-          'Doe'
-        );
-      });       
+      expect(assignProject).toHaveBeenCalledWith(
+        'project123',
+        'member123',
+        'unAssign',
+        'Jane',
+        'Doe'
+      );
```
Comment (stephan-noel) [inline src/components/Projects/Members/Member/Member.test.jsx:119]: nit: It's probably preferable to refer to where the values come from than hardcoding the values again.  I think it helps with communicating intention and maintenance.

**Review by stephan-noel — APPROVED** [review]
A lot of my comments, if not all, are probably out of scope but some can maybe be a quick win.

**Review by this-journey — APPROVED** [review]
Hello! I've attached a video of your functionality working for the owner account.  Great job!  I tested it in both the owner account and the admin account.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/93569791/a2e5cd1d-ed49-409a-820a-6a2d11f05ce4

**Pull request merged**